### PR TITLE
Unify synaptic recordable names for PSC models

### PR DESCRIPTION
**Pull request by Silmathoron**
This is a fix for issue #501 
The PR sets recordable names for synaptic currents as ``"I_syn_ex"`` or ``"I_syn_in"`` for:
* ``amat2_psc_exp`` (added recordables)
* ``hh_psc_alpha``
* ``iaf_psc_alpha``
* ``iaf_psc_exp``

and removed ``"I_ex``, ``"I_in"``, ``"input_currents_ex"``, ``"input_currents_in"`` from ``nest_names``.

### CODE REVIEW

File: models/amat2_psc_exp.h [hunk 1]
```diff
@@ -350,6 +350,16 @@ class amat2_psc_exp : public Archiving_Node
   {
     return S_.V_th_v_;
   }
+  double
```
Comment (heplesser) [inline models/amat2_psc_exp.h]: Insert and empty line before the function definition. Also, I would suggest to add an `inline`.

File: models/amat2_psc_exp.h [hunk 2]
```diff
@@ -350,6 +350,16 @@ class amat2_psc_exp : public Archiving_Node
...
+  get_I_syn_ex_() const
+  {
+    return S_.I_syn_ex_;
+  }
+  double
```
Comment (heplesser) [inline models/amat2_psc_exp.h]: Insert and empty line before the function definition. Also, I would suggest to add an `inline`.

File: models/iaf_psc_alpha.h [hunk 3]
```diff
@@ -310,12 +310,12 @@ class iaf_psc_alpha : public Archiving_Node
     return V_.weighted_spikes_in_;
   }
   double
-  get_input_currents_ex_() const
+  get_I_syn_ex_() const
```
Comment (heplesser) [inline models/iaf_psc_alpha.h]: Added empty line and `inline`.

File: models/iaf_psc_alpha.h [hunk 4]
```diff
@@ -310,12 +310,12 @@ class iaf_psc_alpha : public Archiving_Node
...
   {
     return S_.y1_ex_;
   }
   double
```
Comment (heplesser) [inline models/iaf_psc_alpha.h]: Added empty line and `inline`.

File: models/iaf_psc_exp.h [hunk 5]
```diff
@@ -313,12 +313,12 @@ class iaf_psc_exp : public Archiving_Node
     return V_.weighted_spikes_in_;
   }
   double
```
Comment (heplesser) [inline models/iaf_psc_exp.h]: Added empty line and `inline`.

File: models/iaf_psc_exp.h [hunk 6]
```diff
@@ -313,12 +313,12 @@ class iaf_psc_exp : public Archiving_Node
...
-  get_input_currents_ex_() const
+  get_I_syn_ex_() const
   {
     return S_.i_syn_ex_;
   }
   double
```
Comment (heplesser) [inline models/iaf_psc_exp.h]: Added empty line and `inline`.

**Comment by heplesser** [comment]
:+1: from me. @jhnnsnk , would you take another quick look?

**Comment by DimitriPlotnikov** [comment]
Reasonable unification. Also OK from my side. @heplesser since this can break somebody's code, should one notify malinglist?

**Comment by heplesser** [comment]
@DimitriPlotnikov Good idea to announce!

**Pull request merged**